vtm/src/org/oscim/theme/XmlThemeBuilder.java

```diff
@@ -1299,9 +1299,9 @@ private void tagTransform(String localName, Attributes attributes) {
         }
 
         if (v == null && matchValue == null) {
-            mTransformKeyMap.put(matchKey, k);
+            mTransformKeyMap.put(k, matchKey);
         } else {
-            mTransformTagMap.put(new Tag(matchKey, matchValue), new Tag(k, v));
+            mTransformTagMap.put(new Tag(k, v), new Tag(matchKey, matchValue));
```

Comment:
Oh you reversed the storage, that's why then need to reverse all implementation as well.

Though cannot understand the reason for this change..
Normally need to find the match from the key, that's how is declared in themes and expected to be used.